src/css/base/typography.scss

```diff
@@ -107,26 +119,25 @@ h5 {
 
 p {
   font-family: $font-serif;
-  font-size: $serif-s4;
+  font-size: $serif-s5;
   line-height: $leading-body;
-
-  @include media($medium-screen) {
-    font-size: $serif-s5;
-  }
 }
 
-.page-dek {
-  color: $color-mediumgray;
+.usa-content.content h1 + p, // scope to docs
```

Comment:
the issue with this is that it's difficult to add special classes to the docs markdown, so it's more reliable to do this via adjacency